src/js/components/Drop/index.d.ts

```diff
@@ -9,7 +9,7 @@ export interface DropProps {
     left?: 'left' | 'right';
   };
   elevation?: ElevationType;
-  onClickOutside?: (...args: any[]) => any;
+  onClickOutside?: (this: Document, ev: MouseEvent) => any;
```

Comment:
Verified on the Drop 'Multiple' story, and confirm it works. 
Should the `any` be changed to `void`?

Reply:
Fixed in latest revision.